src/v/cloud_topics/level_one/control_plane.cc

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2025 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "cloud_topics/level_one/control_plane.h"
+
+#include "cloud_topics/logger.h"
+#include "cluster/controller.h"
+#include "cluster/topics_frontend.h"
+#include "cluster/types.h"
+#include "model/fundamental.h"
+#include "model/namespace.h"
+
+#include <seastar/coroutine/as_future.hh>
+
+using namespace std::chrono_literals;
+
+namespace experimental::cloud_topics::l1 {
+
+class control_plane::impl {
+    // Intentionally under the ss::sstring SSO size.
+    static constexpr model::topic_view domain_topic_name = model::topic_view{
+      std::string_view{"ct_l1_domain"}};
+
+public:
+    explicit impl(cluster::controller* controller)
+      : _controller(controller) {}
+
+    ss::future<> start() {
+        _as = {};
```

Comment:
why abort source is reset here? is it expected that start/stop will be called more than once?

Reply:
It's just for duality. Start stop should only be called once. I will remove in a followup